lua/toggleterm/terminal.lua

```diff
@@ -220,7 +220,11 @@ end
 ---@private
 function Terminal:__spawn()
   local cmd = self.cmd or config.get("shell")
-  cmd = fmt("%s;#%s#%d", cmd, term_ft, self.id)
+  if vim.fn.has("win32") then
```

Comment:
Similar to the other comment if we determine once at the start what os someone is on then we can just use a separator variable although I think at this point `fmt` isn't the right way to do this and it should just be strings or table.concat to string 🤷🏾 just that '%s%s%s` etc is meh but not a big deal really fmt or not
```lua
--- at the top-ish
local is_windows = has('win32') -- no need to check this more than once
local cmd_sep = ... -- from other comment

--- re-use in separator here as well 🤷🏾 
cmd = ...
```